src/pages/pool/create.vue

```diff
@@ -59,16 +65,31 @@ const {
   setActiveStep,
   hasInjectedToken,
   seedTokens,
-  totalLiquidity
+  totalLiquidity,
+  hasRestoredFromSavedState,
+  importState
 } = usePoolCreation();
 const { upToLargeBreakpoint, upToSmallBreakpoint } = useBreakpoints();
+const { removeAlert } = useAlerts();
 
 onMounted(() => {
+  removeAlert('return-to-pool-creation');
   if (accordionWrapper.value) {
     anime.set(accordionWrapper.value, {
       opacity: 0
     });
   }
+
+  let previouslySavedState = lsGet(
```

Comment:
I think this code should go in usePoolCreation so it's alongside the saveState code which is similar. Then the key and version doesn't need to be known outside usePoolCreation.

Reply:
unfortunately this cannot be the case since this logic needs to be inside create.vue. If called inside usePoolCreation, it means all the other usages of usePoolCreation in the other children components will make this call again and conflict with the restore